lxd/network/acl/acl_ovn.go

```diff
@@ -1093,8 +1096,7 @@ func ovnParseLogEntry(input string, prefix string) string {
 		return ""
 	}
 
-	// Parse the timestamp.
-	logTime, err := time.Parse(time.RFC3339, fields[0])
+	logTime, err := parseLogTime(timestamp, fields)
```

Comment:
what does `fields[0]` when the ovn log comes from `ovnParseLogEntriesFromJournald`?